<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-01-27 14:31:50.139 ***</b>
=ERROR REPORT==== 27-Jan-2025::14:31:50.139256 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.246698.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1975}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1936}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-01-27 14:31:50.139 ***</b>
=CRASH REPORT==== 27-Jan-2025::14:31:50.139680 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.246703.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2428)
    ancestors: [<0.246698.0>]
    message_queue_len: 0
    messages: []
    links: [<0.246698.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 594
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-01-27 14:32:30.215 ***</b>
=ERROR REPORT==== 27-Jan-2025::14:32:30.214896 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.251048.0&gt;,
                               {#Ref&lt;0.2025474985.3668967425.78981&gt;,1004299,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.2025474985.3668967425.78981&gt;,1004299,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-01-27 14:32:30.215 ***</b>
=CRASH REPORT==== 27-Jan-2025::14:32:30.215149 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.251069.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.2025474985.3668967425.78981&gt;,1004299,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [&lt;0.251048.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 414
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2025-01-27 14:37:12.878 ***</b>
=CRASH REPORT==== 27-Jan-2025::14:37:12.878627 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.256883.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.2025474985.3670278145.148357&gt;,5408,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1696)
    ancestors: [&lt;0.256881.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28117
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-27 14:37:22.441 ***</b>
=ERROR REPORT==== 27-Jan-2025::14:37:22.441158 ===
** Generic server <0.258167.0> terminating 
** Last message in was {nodedown,'peer_down_crash-410435-11901@4d062f6fb1f6'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-410435-11901",
                              peer_down => crash},
                            'peer_down_crash-410435-11901@4d062f6fb1f6',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,50,46,49,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-410435-11901",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDRkMDYyZjZmYjFmNgAD8HcAAAAAZ5eILA=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-410435-11901@4d062f6fb1f6'}},
                            {<0.258153.0>,
                             #Ref<0.2025474985.3663462402.157036>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-410435-11901@4d062f6fb1f6'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2025-01-27 14:37:22.441 ***</b>
=CRASH REPORT==== 27-Jan-2025::14:37:22.441560 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258167.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-410435-11901@4d062f6fb1f6'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2478)
    ancestors: [<0.258153.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1497
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-27 14:37:23.454 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:22.453000 ===
Process <0.92.0> on node 'peer_SUITE-shutdown_stop-522946-11901@4d062f6fb1f6' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-27 14:37:23.815 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:23.815199 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-410531-11901@4d062f6fb1f6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-27 14:37:23.880 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:23.880138 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-522850-11901@4d062f6fb1f6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-27 14:37:39.552 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:38.551351 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-400965-11901@4d062f6fb1f6' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-27 14:37:39.972 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:39.972299 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-410819-11901@4d062f6fb1f6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-27 14:37:40.034 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:40.034223 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-410755-11901@4d062f6fb1f6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-27 14:37:42.223 ***</b>
=ERROR REPORT==== 27-Jan-2025::14:37:42.223232 ===
** Generic server <0.258519.0> terminating 
** Last message in was {'EXIT',<0.258516.0>,
                               {#Ref<0.2025474985.3662675972.193264>,1569607,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-401349-11901",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-401349-11901@4d062f6fb1f6',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,50,46,49,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-401349-11901",
                                     "-user","peer"],
                                    #Port<0.3132>,undefined,<<>>,running,
                                    {<0.258516.0>,
                                     #Ref<0.2025474985.3662675972.193340>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.2025474985.3662675972.193264>,1569607,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-27 14:37:42.223 ***</b>
=CRASH REPORT==== 27-Jan-2025::14:37:42.223650 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258519.0>
    registered_name: []
    exception exit: {#Ref<0.2025474985.3662675972.193264>,1569607,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.258516.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3132>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4537
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-27 14:37:42.778 ***</b>
=ERROR REPORT==== 27-Jan-2025::14:37:42.777886 ===
** Generic server <0.258554.0> terminating 
** Last message in was {'EXIT',<0.258515.0>,
                           {#Ref<0.2025474985.3670278145.167683>,2124960,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "ORBPXSAPNJXZUIXAFJXA","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@4d062f6fb1f6.2025-01-27_14.20.43/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-401381-11901",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@4d062f6fb1f6.2025-01-27_14.20.43/make_test_dir.stdlib_test.logs/run.2025-01-27_14.21.19/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-401381-11901",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.6/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-401381-11901@4d062f6fb1f6',
                                    "/usr/local/lib/erlang-26.2.5.6/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-401381-11901",
                                     "-setcookie","ORBPXSAPNJXZUIXAFJXA",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@4d062f6fb1f6.2025-01-27_14.20.43/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-401381-11901",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@4d062f6fb1f6.2025-01-27_14.20.43/make_test_dir.stdlib_test.logs/run.2025-01-27_14.21.19/log_private/",
                                     "-user","peer"],
                                    #Port<0.3156>,undefined,<<>>,running,
                                    {<0.258515.0>,
                                     #Ref<0.2025474985.3663462402.161007>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.2025474985.3670278145.167683>,2124960,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-27 14:37:42.778 ***</b>
=CRASH REPORT==== 27-Jan-2025::14:37:42.778353 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258554.0>
    registered_name: []
    exception exit: {#Ref<0.2025474985.3670278145.167683>,2124960,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.258515.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3156>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4212
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-27 14:37:43.199 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:42.198664 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-523778-11901@4d062f6fb1f6' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-27 14:37:43.240 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:43.240144 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-420100-11901@4d062f6fb1f6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-27 14:37:43.588 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:43.588228 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-401445-11901@4d062f6fb1f6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-27 14:37:43.605 ***</b>
=WARNING REPORT==== 27-Jan-2025::14:37:43.605158 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-411267-11901@4d062f6fb1f6'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-27 14:59:24.519 ***</b>
=ERROR REPORT==== 27-Jan-2025::14:59:24.519643 ===
** Generic server <0.335449.0> terminating 
** Last message in was {'EXIT',<0.335446.0>,
                               {#Ref<0.2025474985.3701211137.213967>,350568,
                                [{child1,<0.335455.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.335449.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.335455.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458431],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.2025474985.3701211137.213967>,350568,
    [{child1,<0.335455.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-27 14:59:24.520 ***</b>
=ERROR REPORT==== 27-Jan-2025::14:59:24.519645 ===
** Generic server <0.335448.0> terminating 
** Last message in was {'EXIT',<0.335446.0>,
                               {#Ref<0.2025474985.3701211137.213967>,350568,
                                [{child1,<0.335455.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.335448.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.335454.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458431],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.2025474985.3701211137.213967>,350568,
    [{child1,<0.335455.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-27 14:59:24.520 ***</b>
=CRASH REPORT==== 27-Jan-2025::14:59:24.520050 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.335449.0>
    registered_name: []
    exception exit: {#Ref<0.2025474985.3701211137.213967>,350568,
                     [{child1,<0.335455.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.335446.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1027
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-27 14:59:24.520 ***</b>
=CRASH REPORT==== 27-Jan-2025::14:59:24.520162 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.335448.0>
    registered_name: []
    exception exit: {#Ref<0.2025474985.3701211137.213967>,350568,
                     [{child1,<0.335455.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.335446.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1010
  neighbours:


</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2025 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Mon Jan 27 2025 14:21:19<!--/date--><br />
</div>
</center>
</body>
</html>
